certified-connectors/SignHost/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1044 @@
+        },
+        "Link": {
+          "type": "string",
+          "description": "Url containing the link to the file."
+        }
+      }
+    },
+    "FileMetaData": {
+      "type": "object",
+      "properties": {
+        "DisplayOrder": {
+          "type": "integer",
+          "description": "With what order number we'll display the file to the signer"
+        },
+        "DisplayName": {
+          "type": "string",
+          "description": "With what name we'll display the file to the signer"
+        },
+        "SetParaph": {
+          "type": "boolean",
+          "description": "Places a copy of the signer's scribble image on the bottom right of every page where no signature is present.\nNote: due to the nature of advanced or qualified digital signatures, paraphs are merely a cosmetic addition.\n"
+        },
+        "Signers": {
+          "type": "object",
+          "description": "array.\nEach key should be a valid signer id.\n",
+          "additionalProperties": {
+            "type": "object",
+            "properties": {
+              "FormSets": {
+                "type": "array",
+                "description": "List of formset keys to be assigned to this signer.",
+                "items": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "FormSets": {
+          "type": "object",
+          "description": "object.\nThe key of the map will be the formset name.\nThe value will be the formset definition\n",
+          "additionalProperties": {
+            "type": "object",
+            "description": "object\nThe key of the map will be the field name.\nThe value will be the definition of the field.\n",
+            "additionalProperties": {
+              "type": "object",
+              "properties": {
+                "Type": {
+                  "type": "string",
+                  "enum": [
+                    "Seal",
+                    "Signature",
+                    "Check",
+                    "SingleLine"
+                  ],
+                  "description": "Field type to create.\n* Seal is not yet implemented, this will specify the properties of a seal.\n* Signature, specifies a signature field\n* Check, specifies a checkbox. You'll have to set the `value` property\n* SingleLine, specifies a single line textbox\n"
+                },
+                "Location": {
+                  "type": "object",
+                  "description": "Specify where the field should be placed within the document.",
+                  "properties": {
+                    "Search": {
+                      "type": "string",
+                      "description": "The text to search in the pdf document to use as the position for the field. For example `{{Signer1}}`."
+                    },
+                    "Occurence": {
```

Comment:
can you please correct this spelling "occurence"?

Reply:
Removed the property completely as it was unused